apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -16,6 +17,27 @@ export const RowType = {
   STUDENT: "studentRow",
 };
 
+// Constants around moving students to another section.
+// OTHER_TEACHER - value determines whether students will be moved to a section owned by a different teacher
+// COPY_STUDENTS - value determines whether students will be copied to the new section or moved (and subsequently removed from current section)
+export const OTHER_TEACHER = "otherTeacher";
+export const COPY_STUDENTS = "copy";
+
+/** Initial state for manageStudents.transferData redux store.
+  * studentIds - student ids selected to be moved to another section
```

Comment:
For multi-line comments we follow the style shown here, with comment starting on the next line, and aligning with the first `*`, and ending with ` */` like this:
```
/**
 * Initial state...
 * ...
 */
```
I know it's no fun to receive arbitrary style corrections, but we believe it'll make our code more readable if we follow our style guide closely.